app/Http/Controllers/Frontend/DevController.php

```diff
@@ -33,50 +36,70 @@ public function renderUpdateApp(int $appId): View {
         ]);
     }
 
-    public function renderCreateApp(): View {
+    public function renderCreateApp(): View
+    {
         return view('dev.apps-edit', [
             'title' => 'Anwendung erstellen', //ToDo Übersetzen
             'app'   => null
         ]);
     }
 
-    public function updateApp(int $appId, Request $request): RedirectResponse {
+    public function updateApp(int $appId, Request $request): RedirectResponse
+    {
         $validated = $request->validate([
-                                            'name'     => ['required', 'string'],
-                                            'redirect' => ['required', 'string'],
-                                        ]);
+            'name'                   => ['required', 'string'],
+            'redirect'               => ['required', 'string'],
+            'enable_webhooks'        => ['nullable'],
+            'authorized_webhook_url' => ['nullable', 'url'],
```

Comment:
```suggestion
            'authorized_webhook_url' => ['nullable', 'url', 'starts_with:https'],
```

Maybe this is one option to force https. (untested)

Reply:
i think making this configurable would be better, to allow easier local testing using an http proxy